test/spec/spec.controller.js

```diff
@@ -20,6 +20,11 @@ describe('lib/base-controller', () => {
     sinon.stub(BaseController.prototype, 'locals').returns({foo: 'bar'});
   });
 
+  it('sets the correct error class to the instance', () => {
+    controller = new Controller({});
+    controller.Error.should.be.equal(require('../../lib/error'));
```

Comment:
IMO it's ok when a) the file is short enough to all be on screen at once, and b) the code with the `require` is always invoked immediately and synchronously - i.e. is not inside a function call.

It's generally good practice to ensure an app does all of it's `require` calls in the first event loop.